internal/tiltfile/tiltfile_state.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/windmilleng/tilt/internal/model"
 )
 
+// TODO(dmiller) this needs to add all of the files in the directory as dependencies
+const helmFunc = `
+def helm(path):
+	return local("helm template " + path)
```

Comment:
do we also need a way to mark the path as read, so it registers as a config file?

Reply:
Yeah see TODO, and @dbentley's comment.